pkg/storage/pebble.go

```diff
@@ -920,6 +920,18 @@ func DefaultPebbleOptions() *pebble.Options {
 		l.EnsureDefaults()
 	}
 
+	// These size classes are a subset of available size classes in jemalloc[1].
+	// The size classes are used by Pebble for determining target block sizes for
+	// flushes, with the goal of reducing internal fragmentation.
```

Comment:
Do we have any docs / issues we could link here that speaks to why we picked these numbers? That will help our future selves.

Reply:
I can add it into the comment so that it is clear.